moveit_ros/planning/rdf_loader/test/test_rdf_integration.cpp

```diff
@@ -66,6 +66,9 @@ TEST(RDFIntegration, topic_based)
   EXPECT_EQ("gonzo", loader.getSRDF()->getName());
 }
 
+// Failing in Jammy disabling temporarily (TODO (vatanaksoytezer): Fix this and enable again)
+// See https://github.com/ros-planning/moveit2/issues/1156
+/*
```

Comment:
Super nit: Should we disable the test (by adding  `DISABLED_`) rather than commenting it?